### PR TITLE
Documentation correction

### PR DESCRIPTION
**Pull request by Oleg-designer**
ref #560

### CODE REVIEW

**Comment by pzhlkj6612** [comment]
Hi.

The indentation changes in 269153227c6f0229f0dd1723d54312febaf9ad6d doesn't seem to make much sense. What's your purpose about it?

**Comment by Oleg-designer** [comment]
> Hi.
> 
> The indentation changes in [2691532](https://github.com/QuasarApp/CQtDeployer/commit/269153227c6f0229f0dd1723d54312febaf9ad6d) doesn't seem to make much sense. What's your purpose about it?

Hi, i am doing small fixes for doxygen documentation. Yes,  2691532 there is no point in making these changes, it's just for testing.

**Review by pzhlkj6612 — COMMENTED** [review]
In /md/en/Guide.md , there are many modifications about removing a path delimiter from a file path. For example:

```diff
- /home/andrei/Qt/5.15.2/gcc_64/qml/QtQml/WorkerScript.2/plugins.qmltypes
+ /home/andrei/Qt/5.15.2/gcc_64/qml/QtQmlWorkerScript.2/plugins.qmltypes
```

Is it on purpose?

File: md/en/Guide.md [hunk 1]
```diff
@@ -206,261 +204,210 @@ cqtdeployer -bin myexecutable -libDir /PathToMyExtraLibs -recursiveDepth 5 -qmlD
...
-    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuick/Particles.2/qmldir
-    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuick.2/libqtquick2plugin.so
-    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuick.2/plugins.qmltypes
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQmlWorkerScript.2/plugins.qmltypes
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQmlWorkerScript.2/qmldir
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuickParticles.2/libparticlesplugin.so
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuickParticles.2/plugins.qmltypes
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuickParticles.2/qmldir
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuick.2libqtquick2plugin.so
+    Info: copy :/home/andrei/Qt/5.15.2/gcc_64/qml/QtQuick.2plugins.qmltypes
```
Comment (EndrII) [inline md/en/Guide.md:213]: you remove '/' from paths. please revert this changes.

**Review by EndrII — CHANGES_REQUESTED** [review]

**Comment by EndrII** [comment]
> In /md/en/Guide.md , there are many modifications about removing a path delimiter from a file path. For example:
> 
> ```diff
> - /home/andrei/Qt/5.15.2/gcc_64/qml/QtQml/WorkerScript.2/plugins.qmltypes
> + /home/andrei/Qt/5.15.2/gcc_64/qml/QtQmlWorkerScript.2/plugins.qmltypes
> ```
> 
> Is it on purpose?

i think it is typo mistake.

**Review by EndrII — APPROVED** [review]
looks good for me.

**Pull request merged**